backend/src/auth/strategies/pw-reset-token.strategy.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import { PassportStrategy } from '@nestjs/passport'
+import { ExtractJwt, Strategy } from 'passport-jwt'
+import { UserService } from '../../db/services/user.service'
+import { PasswordResetJWTPayload } from '../interfaces/pw-reset-jwt-payload.interface'
+
+/**
+ * @description Strategy for validating long living refresh tokens
+ */
+@Injectable()
+export class PasswordResetTokenStrategy extends PassportStrategy(
+  Strategy,
+  'password-reset-jwt',
+) {
+  constructor(
+    private readonly configService: ConfigService,
+    private readonly userService: UserService,
+  ) {
```

Comment:
```suggestion
  constructor(private readonly configService: ConfigService) {
```

userService is unused?